server/models/services/AtBrowserService

```diff
@@ -0,0 +1,27 @@
+const ModelService = require('./ModelService');
```

Comment:
It looks like this file doesn't have a file extension. `.js` should be added.

Reply:
Ok. `.js` added.